include/votca/tools/globals.h

```diff
@@ -15,10 +15,13 @@
  *
  */
 
-#ifndef __VOTCA_TOOLS_GLOBALS_H
-#define __VOTCA_TOOLS_GLOBALS_H
+#ifndef VOTCA_TOOLS_GLOBALS_H
+#define VOTCA_TOOLS_GLOBALS_H
 
+// Standard includes
 #include <string>
+
+// Local VOTCA includes
 #include <votca/tools/votca_config.h>
```

Comment:
@junghans I'm not quite sure how to adjust cmake so that it find "votca_config.h" after it has been configured.

Reply:
Fixed.